src/Symfony/Component/Form/CallbackTransformer.php

```diff
@@ -11,6 +11,9 @@
 
 namespace Symfony\Component\Form;
 
+/**
+ * @implements DataTransformerInterface<mixed, mixed>
```

Comment:
this could be actually a generic class, by providing actual signatures for the callables